cdc/puller/file_sorter.go

```diff
@@ -415,7 +415,7 @@ func (fs *FileSorter) rotate(ctx context.Context, resolvedTs uint64) error {
 			// file sorter outputs all events in this rotate round.
 			rowCount += 1
 			if rowCount%defaultAutoResolvedRows == 0 {
-				fs.output(ctx, model.NewResolvedPolymorphicEvent(item.entry.CRTs))
+				fs.output(ctx, model.NewResolvedPolymorphicEvent(item.entry.CRTs-1))
```

Comment:
what happened if the current `item.entry.CRTs-1` is equal to the `item.entry.CRTs` of the previous event

Reply:
Events are sorted by commit-ts here, and the previous is guaranteed to be sent to output chan